### DISCUSSION TITLE
How to convert u64 to string

### DISCUSSION THREAD

**Discussion by 0x-j** [aptos-labs/aptos-developer-discussions#189] [Questions] [answered]
### Discord user ID

ctb0k33

### Describe your question in detail.

As title, asked in discord dev-discussion

### What have you tried or looked at?  Or how can we reproduce the error?

N/A

### Which operating system are you using?

macOS

### Which SDK or tool are you using? (if any)

N/A

**Comment by 0x-j** [comment, marked as answer]
Here's how https://github.com/0xaptosj/playground/blob/cca820af1bdc3d2a8855c31aec4b1d16e78beeb4/sources/playground_1.move#L19-L27

**Reply by gregnazario** [reply]
More generally, you can format any types using `string_utils::formatX` or `string_utils::to_string` https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-stdlib/sources/string_utils.move#L12-L21
